### DISCUSSION TITLE
how to properly build for android ?

### DISCUSSION THREAD

**Discussion by moolight-seashell** [slint-ui/slint#6054] [Q&A]
hmm I'm a bit confused at the moment

because cargo-apk seems deprecated replaced by xbuild.

but xbuild seems to be usable only in nightly mode
```
The sysroot can't be built for the Stable channel. Switch to nightly
```

and the xbuild docs looks not very clear to me

**Comment by Enyium** [comment]
At least part of this was addressed [on Reddit](https://www.reddit.com/r/rust/comments/1bej4ge/slint_v15_rust_gui_toolkit_released_with_android/kuv2vq9/).

**Comment by moolight-seashell** [comment]
we have the "x" command everywhere but obviously x does not exist

and i d'ont want to use nightly channel
